project_task_category/models/project_task_category.py

```diff
@@ -11,4 +11,5 @@ class ProjectTask(models.Model):
     categ_id = fields.Many2one(
         'project.category',
         'Category',
+        domain=[('code', '=', 'categ_id')]
```

Comment:
> I suggest that you "copy" project.categ into a new model (project.categ.main?), using prototype inheritance:
class ProjectCategMain(models.Model):
    _name = `project.categ.main`
    _inherit = `project.categ`
This also makes easy to have different ACLs on each Category group: main Categories can be read only for regular users, and Tags Categories can be read-write, for example, to support dynamic reusable keywords.

@dreispt Yes, is what we want to have from the very beginning when we start this matter with the BR modules (we had before our own model `task.category` instead of use `project.category`).

I'll follow your recommendation for a new model `project.category.main` inherited from `project.category`.

Thanks!